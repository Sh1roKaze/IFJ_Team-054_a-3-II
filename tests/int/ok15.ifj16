class Main
{
	static void run()
	{
		int i = 10 / 4;
		double a = 11 / i;
		double b = a / 2;
		ifj16.print("i:" + i + " a:" + a + " b:" + b + "\n");
	}
}
