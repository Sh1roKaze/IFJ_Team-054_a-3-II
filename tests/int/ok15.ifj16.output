i:2 a:5 b:2.5
