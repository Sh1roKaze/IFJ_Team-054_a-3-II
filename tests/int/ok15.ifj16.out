i: a:5.5 b:2.75
